easy_rec/python/utils/convert_rtp_fg.py

```diff
@@ -201,6 +207,34 @@ def convert_rtp_fg(rtp_fg,
       else:
         pipeline_config.feature_config.features.append(feature_config)
       pipeline_config.data_config.input_fields.append(input_field)
+
+      if 'extra_combo_info' in feature:
```

Comment:
The operation `Combo` has been done in FG, why should we convert ComboFeature here? simply treat it as an id_feature.

Reply:
add extra_combo_info in fg doc

Reply:
done